backend/polling.ml

```diff
@@ -261,7 +261,8 @@ let find_poll_alloc_or_calls instr =
   List.rev !matches
 
 let instrument_fundecl ~future_funcnames:_ (f : Mach.fundecl) : Mach.fundecl =
-  if function_is_assumed_to_never_poll f.fun_name then f
+  if !Flambda_backend_flags.disable_poll_insertion ||
+     function_is_assumed_to_never_poll f.fun_name then f
```

Comment:
There is one instance of `function_is_assumed_to_never_poll`
(line 153, outside of the diff) that is not being augmented
with the additional condition. Is that on purpose?